src/Internal/DefaultCacheProvider.php

```diff
@@ -0,0 +1,53 @@
+<?php
```

Comment:
Ignore this class from coverage since we're dealing with multiple paths depending on versions of dependencies